docker-compose.yml

```diff
@@ -70,7 +70,16 @@ services:
         extra_hosts:
             - "dockerhost:${DOCKER_HOST_IP}"
         environment:
-            - PHP_IDE_CONFIG=${PHP_IDE_CONFIG}
+            - PHP_IDE_CONFIG=serverName=laravel
```

Comment:
I havn't tried and never did before, but supposedly it should just replace that text from the env, I see no reason why shouldn't unless it never did! :P